src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messages.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block menu %}
+<span class="label">
+    <span class="icon">{{ include('@WebProfiler/Icon/messages.svg') }}</span>
+    <strong>Messages</strong>
+
+    {% if collector.messages | length > 0 %}
```

Comment:
`collector.messages | length` -> `collector.messages|length`  ... and for other Twig filters too.